operations/routingrules.md

```diff
@@ -0,0 +1,380 @@
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `RoutingRules` | array of [Routing rule update parameters](routingrules.md#routing-rule-update-parameters) | required, max 1000 items | Routing rules to be updated. |
+
+#### Routing rule update parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `RoutingRuleId` | string | required | Unique identifier of the routing rule. |
+| `CompanyId` | [String update value](#string-update-value) | optional | Unique identifier of the [Company](companies.md#company) \(or `null` if should not be updated\). |
+| `CompanyRelation` | [String update value](#string-update-value)| optional | [Company relation](routingrules.md#company-relation) to the routing rule \(or `null` if should not be updated\). |
+| `Applicability` | [String update value](#string-update-value) | required | [Applicability](routingrules.md#applicability) that determines if routing rule apply to all future reservations with this company or travel agency attached or only future reservations that are prepaid in online travel agency (OTA) \(or `null` if should not be updated\). |
+| `ServiceId` | [String update value](#string-update-value) | required | Unique identifier of the [Service](services.md#service) routing rule is assigned to \(or `null` if should not be updated\). |
+| `RouteType` | [String update value](#string-update-value) | required | What should be routed [Route type](routingrules.md#route-type) \(or `null` if should not be updated\). |
+| `SelectedStayItems` | [Selected stay items update parameters](routingrules.md#selected-stay-items-update-parameters) | optional | To which stay items routing rule applies to. Returns only if RouteType value is SelectedStayItems. |
+
+#### String update value
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Value` | string | optional | Value which is to be updated. |
+
+#### String array update value
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Value` | array of string | optional | Value which is to be updated. |
+
+#### Bool update value
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Value` | bool | optional | Value which is to be updated. |
+
+#### Selected stay items update parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Nights` | [Bool update value](#bool-update-value) | optional | Whether routing rule applies to nights \(or `null` if should not be updated\). |
+| `CityTax` | [Bool update value](#bool-update-value) | optional | Whether routing rule applies to city tax \(or `null` if should not be updated\). |
+| `ProductCategoryIds` | [String array update value](#string-array-update-value) | optional | Product categories to which routing rule applies to \(or `null` if should not be updated\). |
```

Comment:
Small point of grammar - if you use "to which" then you don't need the additional "to" on the end, e.g. "Product categories to which the routing rule applies."

Reply:
Thanks for this, I'll utilize it. I stopped commenting because I was just about to redo this 😄